.version

```diff
@@ -1 +1 @@
-4.2.2
\ No newline at end of file
+4.2.3
```

Comment:
If one of the underlying dependencies (iOS and/or Android) had a minor version bump, should this be a minor version bump too? @tonidero @vegaro
Then if PHC gets a minor version bump, the hybrids that use it could too.

Reply:
Hmm right... that would increase the amount of minors/majors in PHC and hybrids, since they would have to be increased for the corresponding minors and majors of iOS and Android, but I think that's probably desired behavior. Currently, that's not implemented though, it would only look at the labels of the PRs since the last release. I think for now, if we want to make this a minor, we can tag the PR with the iOS bump with `feat`, close this PR and rerun the job to create a minor. We would need to update the Changelog as well.

And about how to implement this in the future, we can think about it since I don't think it's trivial, specially the autogeneration of the CHANGELOG.

Reply:
That's an interesting one. I actually think if the public API doesn't change, it should still be a patch. Last time we increased BillingClient to version 4, we released a minor I think, not a major.

I actually found this comment https://github.com/semver/semver/issues/148#issuecomment-21970506 which pretty much summarizes what I think about it.

This is an interesting definition of the problem:

```
It isn't a bug fix/patch so 4.2.3 seems wrong.
It isn't an addition of functionality so 4.3.0 seems wrong.
It is still backwards compatible so 5.0.0 seems wrong.
```

Reply:
Yeah it's tricky. In most cases I agree, but in this one for example we made SK2 the default. If we don't override that in some of the hybrids (not sure if all set it to false / true explicitly) then that's definitely not a patch...

Reply:
ah, in that case yes, it makes sense. Do you want me to take care of rerunning it as a 4.3.0?

Reply:
I can manually change it, I just wanted to hear what your opinions.
Being conservative and doing what I suggested in all cases might be best, even if in some cases a "new feature" doesn't affect the hybrids until it's been exposed. What do you think?

Reply:
Yes, I think it makes sense to be conservative. I think even for purchases-android being updated to Billing Client 5 we would want to release a new major for observer mode folks.